packages/amplify-cli/src/extensions/amplify-helpers/update-amplify-meta.ts

```diff
@@ -2,18 +2,10 @@ import * as fs from 'fs-extra';
 import * as path from 'path';
 import { hashElement } from 'folder-hash';
 import { updateBackendConfigAfterResourceAdd, updateBackendConfigAfterResourceUpdate } from './update-backend-config';
-import { readJsonFile } from './read-json-file';
-import {
-  getAmplifyMetaFilePath,
-  getCurrentAmplifyMetaFilePath,
-  getBackendConfigFilePath,
-  getCurrentBackendConfigFilePath,
-  getBackendDirPath,
-  getCurrentCloudBackendDirPath,
-} from './path-manager';
-
-export function updateAwsMetaFile(filePath, category, resourceName, attribute, value, timeStamp) {
-  const amplifyMeta = readJsonFile(filePath);
+import { JSONUtilities, $TSMeta, pathManager, stateManager } from 'amplify-cli-core';
+
+export function updateAwsMetaFile(filePath, category, resourceName, attribute, value, timestamp) {
```

Comment:
Can't we nix this completely and use statemanager

Reply:
No, because this function is exported in `AmplifyToolkit` and used across other packages. This and the other state manager helpers can be safely removed after the `AmplifyToolkit` interface is fully made into an interface. Interface in `amplify-cli-core`, implementation in `amplify-cli`